common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -761,6 +761,41 @@ colored_emblem_texture_lists = {
 			100 	= "wc_generic_ce_orc_symbol_68.dds"
 			100 	= "wc_generic_ce_orc_symbol_69.dds"
 			100 	= "wc_generic_ce_orc_symbol_70.dds"
+			100 	= "ce_rune_orc_01.dds"
```

Comment:
There's another range with orcish emblems at 1774 line. So they should be duplicated there too with 2000 chance of appearing.

Duplication isn't good stuff, but sadly it's how Paradox made CoAs.

Reply:
Done !